test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -59,6 +60,37 @@ public void EnumeratorCacheIsScopedByAssembly()
             Assert.Equal(expectedCallCount, resourceAssembly2.GetManifestResourceStreamCallCount);
         }
 
+        [Fact]
+        public void ResourceLookupBehaviorUseNameIfNotFound()
```

Comment:
The name should be descriptive of the assertion as well, e.g. `ReturnsNameForMissingStringWhenResourceLookupBehaviorIsUseNameIfNotFound`